src/test/steps/happy-path.ts

```diff
@@ -16,6 +17,12 @@ Given("I've already completed the form using the fixture {string}", async (fixtu
   I.amOnPage(url);
 });
 
+Given('I set the case state to Holding', async () => {
+  iSetTheUsersCaseTo({
+    state: State.Holding,
```

Comment:
very smart but might cause issues down the line, i think this might question the FE more how we save the formdata to session even if it fails in the backend. I think for now its fine.